### ISSUE TITLE
Element has insufficient color contrast

### ISSUE THREAD

**Issue by lfrancis** [forumone/peacecorps-site#395]
https://stage-donate.peacecorps.gov/donate/projects-funds/#issue

Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

```
wcag2aa wcag143 
```

Target: #mainContent > .js-discoverApp.discover > ul:nth-of-type(3) > li:nth-of-type(10) > .column--half.section__content > .t-title--1
HTML:

<h1 class="t-title--1">Water &amp; Sanitation</h1>

Summary:
Fix any of the following:

```
Element has insufficient color contrast of 4.46 (foreground color: #67655d, background color: #e5e1d4, font size: 17.3pt, font weight: normal)
Related Nodes:
    #mainContent > .js-discoverApp.discover > ul:nth-of-type(3) > li:nth-of-type(10)
```
